dgraphql/resolvers/resolver.go

```diff
@@ -820,6 +820,24 @@ func (t *TransactionTrace) ExceptJSON() (*commonTypes.JSON, error) {
 	return nil, nil
 }
 
+type BlockRootMerkle struct {
+	m *pbcodec.BlockRootMerkle
+}
+
+func newBlockRootMerkle(merkleRoot *pbcodec.BlockRootMerkle) *BlockRootMerkle {
```

Comment:
A non-pointer type `BlockRootMerkle` should work here since the field is required, for small object, we try to avoid pointers.

```suggestion
func newBlockRootMerkle(merkleRoot *pbcodec.BlockRootMerkle) BlockRootMerkle {
```

Reply:
Fixed